pkg/cnab/provider/parameters.go

```diff
@@ -58,8 +46,8 @@ func (d *Runtime) loadParameters(claim *claim.Claim, rawOverrides map[string]str
 	return bundle.ValuesOrDefaults(overrides, bun, action)
 }
 
-// loadParameterSets loads parameter values per their parameter set strategies
-func (d *Runtime) loadParameterSets(b *bundle.Bundle, params []string) (valuesource.Set, error) {
+// LoadParameterSets loads parameter values per their parameter set strategies
+func (d *Runtime) LoadParameterSets(params []string) (valuesource.Set, error) {
```

Comment:
I think that this function can be moved out of the pkg/cnab/provider and into pkg/porter. Nothing in pkg/cnab/provider package calls this, and none of the code here relies on being located in this package, right? So we could move it up closer to where it's used (the porter client) and not have to have this in the runtime.

Reply:
Indeed, thank you.  Now moved to `pkg/porter`.  PTAL when convenient.